src/settings/Api.ts

```diff
@@ -3,9 +3,23 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 import { generateUrl, generateOcsUrl } from '@nextcloud/router'
-import { OCSResult, AxiosOCSResult } from 'NC'
 import axios from '@nextcloud/axios'
 
+interface AxiosOCSResult<T> {
```

Comment:
No need for custom interfaces, you could also just use the interface from `@nextcloud/typings`.

Reply:
Ah I didn't know this was available, I only moved the interface into the file.